package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "testium-driver-wd",
-  "version": "2.7.0",
+  "version": "2.5.0",
```

Comment:
This looks like an out-of-date base commit..?

Reply:
thx . rebase mistake. fixing